### ISSUE TITLE
iOS Label Mappings needs update

### ISSUE THREAD

**Issue by cddiggs** [MenelicSoftware/cordova-plugin-brother-label-printer#24]
Printer: QL-820NWB
Label Size: W29H90

I can print the labels from an Android device successfully. However, I get an ERROR_WRONG_LABEL when trying to print from an iOS device. It looks like this plugin is using the Brother SDK version 3.1.1. I wasn't able to find the documentation for that specific SDK version. Based on the documentation from the Brother SDK version 3.1.12, the iOS label mapping are as follows: 

**QL Series:**
Constant | Description
-- | --
@"12mm" | 12mm continuous length label
@"29mm" | 29mm continuous length label
@"38mm" | 38mm continuous length label
@"50mm" | 50mm continuous length label
@"54mm" | 54mm continuous length label
@"62mm" | 62mm continuous length label
@"62mmRB" | 62mm continuous length label (red and black)
@"102mm" | 102mm continuous length label
@"103mm" | 103mm continuous length label
@"17mmx54mm" | 17mm x 54mm label
@"17mmx87mm" | 17mm x 87mm label
@"23mmx23mm" | 23mm x 23mm label
@"29mmx42mm" | 29mm x 42mm label
@"29mmx90mm" | 29mm x 90mm label
@"38mmx90mm" | 38mm x 90mm label
@"39mmx48mm" | 39mm x 48mm label
@"52mmx29mm" | 52mm x 29mm label
@"54mmx29mm" | 54mm x 29mm label
@"60mmx86mm" | 60mm x 86mm label
@"62mmx29mm" | 62mm x 29mm label
@"62mmx100mm" | 62mm x 100mm label
@"102mmx51mm" | 102mm x 51mm label
@"102mmx152mm" | 102mm x 152mm label
@"103mmx164mm" | 103mm x 164mm label

In other words, we need to update the iOS label mappings in www/printer.js. (i.e. update from '29mmx90' to '29mmx90mm'. 

Thanks for supporting this plugin!

**Comment by arcadius** [comment]
Thanks @cddiggs  for the pull request.

Please is this change working for you for both Android and iOS?

**Comment by cddiggs** [comment]
@arcadius Yes, I can guarantee this is working on Android and iOS for the 29mmx90mm label size. I'm not able to verify the other label sizes with fixed heights, but I expect those will all have the same issue.

**Comment by arcadius** [comment]
I will give it a try and merge asap

**Comment by arcadius** [comment]
Thank you very much @cddiggs

**Comment by cpgb85** [comment]
Is this fixed yet?
